01-introduction.Rmd

```diff
@@ -41,20 +56,30 @@ This book aims to go beyond teaching methods and code: by the end of it you shou
 Fast-forward two decades and things have progressed rapidly.
-Anyone with access to a laptop with ~4GB RAM can realistically expect to be able to install and run software for geocompuation on publicly accessible datasets, which are more widely available than ever before (as we'll see in Chapter \@ref(read-write)).^[
+Anyone with access to a laptop with ~4GB RAM can realistically expect to be able to install and run software for geocompuation on publicly accessible datasets, which are more widely available than ever before (as we will see in Chapter \@ref(read-write)).^[
 A laptop with 4GB running a modern operating system such as Ubuntu 16.04 onwards should also be able to reproduce the contents of this book.
 A laptop with this specification or above can be acquired second-hand for ~$100 in many countries nowadays, reducing the financial/hardware barrier to geocomputation far below the levels in operation in the early 2000s, when high-performance computers were unaffordable for most people.
 ]
 Unlike early works in the field all the work presented in this book is reproducible using code and example data supplied alongside the book, in R packages such as **spData**, the installation of which is covered in Chapter \@ref(spatial-class).
 
-Reproducible geographic research is related to Geographical Data Science (GDS).
-This recent concept essentially combines 'data science' with GIS and, like geocomputation, can be defined in comparison with GIS (see Table \@ref(tab:gdsl)).
-The focus on reproducibility and a command-line interface in this book is aligned with GDS.
-
-```{r gdsl, echo=FALSE, message=FALSE}
-d = readr::read_csv("extdata/gis-vs-gds-table.csv")
-knitr::kable(x = d, caption = "Differences in emphasis between the fields of Geographic Information Systems (GIS) and Geographic Data Science (GDS).")
-```
+One should be aware of the fact that Geocomputation has quite a few close cousins including Geographic Information Science (GIScience), Geomatics, Geoinformatics, Spatial Information Science, Geoinformation Engineering [@longley_geographic_2015], and of late also Geographic Data Science (GDS).
+All these terms share the scientific approach to study geographic data via geographic information systems, and differ mostly in their origin and major field of application, respectively.
+For example, GDS emphasizes the need for reproducible, automated data analysis of (big) geographic data by means of data science methods.
+<!-- @Robin, please rephrase if the distinguition between GDS and geocomputation is not correct --> 
+In fact, GDS can be used as a synonym for Geocomputation with the only difference that Geocomputation has a slightly larger focus on the applied geographic analysis and GDS on creating novel techniques suitable for statistical geoprocessing.
+Overall, the focus on reproducibility via a command-line interface to do geographic analysis are recurring themes of this book.
+
+<!-- 
```

Comment:
An interesting comment - will look to opportunities to integrate while prioritising the need to keep the chapter short and compellingly readable.